modules/services/repositories/src/main/java/au/com/shiftyjelly/pocketcasts/repositories/download/DownloadManagerImpl.kt

```diff
@@ -470,6 +484,10 @@ class DownloadManagerImpl @Inject constructor(
     }
 
     private fun updateNotification() {
+
+        // Don't show these notifications on wear os
+        if (Util.isWearOs(context)) return
```

Comment:
I don't have a good feel for how to use notifications on the watch yet. It feels like we shouldn't use them as much as we do in the app, so I've disabled this one for now (also, it's not working on the watch currently), but perhaps we'll want to add it back later. Let me know if you have any thoughts here.

Reply:
Looks like we're not setting up notification channels for the wear os app inside `PocketCastsWearApplication` -> `setupApp()` : `notificationHelper.setupNotificationChannels()`

I can see notification by adding it:

<img src="https://user-images.githubusercontent.com/1405144/223957715-547622f5-642f-4929-8fac-1772375f4caa.png" width=200/>

Reply:
👍 Thanks for that. I'll make this update in a separate PR since this one is already way too big.